sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/DefaultProjectIssues.java

```diff
@@ -41,16 +43,18 @@ public DefaultProjectIssues(IssueCache cache) {
 
   @Override
   public Iterable<Issue> issues() {
-    return Iterables.transform(
-      Iterables.filter(cache.all(), new ResolvedPredicate(false)),
-      new IssueTransformer());
+    return StreamSupport.stream(cache.all().spliterator(), false)
+      .filter(new ResolvedPredicate(false))
```

Comment:
Seems a waste to recreate the predicate instance for each object. Since instances of `ResolvedPredicate` are immutable, it could be a good idea to make them constants